src/Generator/MutatorAccessorClassGeneratorAbstract.php

```diff
@@ -156,4 +163,224 @@ protected function generateEnumValidation(Field $root, string $baseNamespace): ?
 
         return $this->builder->if($ifCondition, [$ifStmt]);
     }
+
+    protected function generateMinimumValidation(Field $root): ?Stmt
+    {
+        $minimum = $root->getMinimum();
+        if ($minimum === null) {
+            return null;
+        }
+
+        $this->addImport(RequestValidationException::class);
+
+        $propertyVar = $this->builder->var($root->getPhpVariableName());
+        $operator    = $root->isExclusiveMinimum() === true ? '<=' : '<';
+
+        $ifCondition = $this->builder->compare(
+            $propertyVar,
+            $operator,
+            $this->builder->val($minimum)
+        );
+
+        $exceptionMessage = $this->builder->funcCall(
+            'sprintf',
+            [
+                'Invalid %s value. Given: `%s`, cannot be %s than %s',
+                $root->getName(),
+                $propertyVar,
+                $operator,
+                $minimum,
+            ]
+        );
+
+        $ifStmt = $this->builder->throw('RequestValidationException', $exceptionMessage);
```

Comment:
Most of the code is duplicated between these methods with just some minor details being different, like exception messages.
If we would have a FieldValidation object in Field, we can have a switch and reuse some parts of the code. This is not a must, just this class could have been prettier.

Reply:
agree, I'll refactor it np

Reply:
done, please check